src/app/components/cds/charts/NumberWidget.js

```diff
@@ -10,15 +10,24 @@ const NumberWidget = ({
   unit,
 }) => (
   <div className={styles.numberWidgetWrapper} style={{ backgroundColor: color }}>
-    <div className="typography-subtitle2">
+    <div className={styles.numberWidgetTitle}>
       {title}
     </div>
-    <div className={styles.numberWidgetItemCount}>
-      {itemCount} <span className={styles.numberWidgetUnit}>{!itemCount || itemCount === '-' ? null : unit} </span>
-    </div>
-    <div className={styles.numberWidgetContextText}>
-      {contextText}
-    </div>
+    { itemCount && itemCount !== '-' &&
```

Comment:
might be able to condense this to 
```
{ itemCount &&
      <div className={styles.numberWidgetItemCount}>
        {itemCount} { unit && itemCount !== '-' && <span className={styles.numberWidgetUnit}>{unit}</span> }
      </div>
    }
{ contextText &&
      <div className={styles.numberWidgetContextText}>
        {contextText}
      </div>
    }
```
Otherwise looks good!